test/mocks/OkexExchangeScenarioStepBuilder.ts

```diff
@@ -351,24 +350,30 @@ export class OkexExchangeScenarioStepBuilder {
     // Update in-flight
     if (wasFundTransferExpected) {
       if (wasTransferWithdraw) {
+        console.log("Mock: fetchWithdrawals()")
         this.exchangeMockObject.fetchWithdrawals.mockImplementationOnce(
-          ({ address, amountInSats }) => {
+          (code, since, limit, params) => {
             const args = {
-              address: address,
-              amountInBtc: sat2btc(amountInSats),
+              address: params.address,
+              amountInBtc: sat2btc(params.amountInSats),
               status: FundTransferStatus.Ok, // <-- Always successful for now
             }
+            console.log(
```

Comment:
should be using pino logger instead of console.log for consistency

Reply:
Yeah I'll clean them all up, just used to contrast with pino for temporary stuff. and sometimes get committed before clean up. thx

Reply:
logger.debug or logger.trace can be used for that.